gax/src/test/java/com/google/api/gax/tracing/TracedClientStreamingCallableTest.java

```diff
@@ -165,7 +166,7 @@ public void testRequestNotify() {
       }
 
       this.responseObserver = responseObserver;
-      this.callContext = callContext;
+      this.callContext = context;
```

Comment:
...what? was this just a strange bug that never caused a problem before??

Reply:
It is a missing test, an a corresponding issue was created for it https://github.com/googleapis/gax-java/issues/687